serverless/sql-databases/api-cli/import-data-to-serverless-sql-databases.mdx

```diff
@@ -0,0 +1,209 @@
+    ```
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/serverless-db/databases) of your database.
+    </Message>
+
+4. Enter the database password when prompted. 
+
+    <Message type="note">
+    The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+5. When finished, make sure your data is stored in your new database by [connecting to it](/serverless/sql-databases/how-to/connect-to-a-database/), and performing a query.
+
+### Using pgAdmin
+
+#### Prerequisites
+
+To complete this procedure, you must have:
+
+- Installed [pg_Admin](https://www.pgadmin.org/).
+
+- Ensured your PGAdmin version is compatible with the PostgreSQL version of the database you will import your data into. Currently, pgAdmin 7.8 or newer is required to fully work with PostgreSQL 16.
+
+#### Downloading the backup on your local machine
+
+1. Open pgAdmin.
+
+2. Connect to your existing database server by selecting it in the left-hand menu.
+
+3. In the **Databases** submenu, connect to the database you want to export by selecting it.
+
+4. In the left-hand menu, right-click the database you want to export, and select **Backup**.
+
+5. Enter a name for the backup file (e.g. `my-backup`), and keep the default **format** and **compression** options.
+
+6. Click **Backup**. The backup creation process starts, and you can follow its progress by clicking **View processes**.
+
+7. When finished, make sure your backup is downloaded on your local machine. By default, the backup location is `Users/{username}/my-backup` on MacOS.
+
+#### Loading your data in your Serverless SQL Database
+
+1. Register your new database in pgAdmin. Refer to the [How to connect to a Serverless SQL Database](/serverless/sql-databases/how-to/connect-to-a-database/#how-to-set-up-credentials) documentation for more information.
+
+2. Connect to your database server by selecting it in the left-hand menu.
+
+3. Select your database in the **Databases* submenu.
+
+4. In the left-hand menu, right-click the name of your new database, then select **Restore**.
+
+5. Enter the file name of your backup (e.g. `my-backup`), and keep the default options.
+
+6. Access the **Data options** tab, and enable the **Do not save - Owner** and **Do not save - Privileges** options.
+
+7. (Optional) If your new database is not empty, access the **Query Options** tab, and enable the **Clean before restore** option. 
+
+    <Message type="tip">
+    This option ensures your database is emptied before loading data to avoid potential conflicts with existing data (such as table names or primary key constraints).
```

Comment:
```suggestion
    This option ensures that your database is emptied before loading data to avoid potential conflicts with existing data (such as table names or primary key constraints).
```